documentation/developer_documentation/RELEASE_CHECKLIST.md

```diff
@@ -0,0 +1,28 @@
+Forest follows a fixed, quaterly release schedule. On the last Friday of each
```

Comment:
In general, releases just before the weekend should be avoided. When we go to production, and there's a severe issue missed, it's better to have the entire team available (and not one poor guy who has to fix it over the weekend). Plus, it's better to give folks some slack in case of unexpected problems. I recommend Wednesdays. Or even no fixed day, we can schedule a release "week".

The last day of a quarter is also a bit tight for no apparent reason. For example, it would not work for us in Q4 because of ChainSafe days. I'd phrase it "quarterly" according to a schedule we can contrive and manage without significant overhead. 

Later on, we can always make it tight if there's a need. What do you think?

Reply:
In my mind, not releasing on Fridays is like the widespread (and incorrect) advice to not push to production on Fridays. The release date doesn't matter, though, so I'll change the wording to say the last week in each quarter.